### PR TITLE
Add MosaicML Composer example

### PR DESCRIPTION
**Pull request by tcapelle**
Add a simple MNIST example on using composer and adding a custom callback to log a `wandb.Table`

### CODE REVIEW

File: colabs/mosaicml/MosaicML_Composer_and_wandb.ipynb [hunk 1]
```diff
@@ -0,0 +1,765 @@
+{
```
Comment (morganmcg1) [inline colabs/mosaicml/MosaicML_Composer_and_wandb.ipynb:1]: <p>I would add some configs here too, like batch size, model name etc etc unless these get picked up by the logger itself?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (morganmcg1) [inline colabs/mosaicml/MosaicML_Composer_and_wandb.ipynb:1]: <p>I would push this to the bottom in an "advanced" section. To start I think we should just showcase a super simple and quick to understand example of how the logger works. Then we can show off more advanced features like this custom callback for Tables</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (morganmcg1) [inline colabs/mosaicml/MosaicML_Composer_and_wandb.ipynb:1]: <p>We have this code twice?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (morganmcg1) [inline colabs/mosaicml/MosaicML_Composer_and_wandb.ipynb:1]: <p>Image is broken for me (in ReviewNB at least)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (tcapelle) [reply colabs/mosaicml/MosaicML_Composer_and_wandb.ipynb:1]: good catch, it appears they don't get logged

Comment (tcapelle) [reply colabs/mosaicml/MosaicML_Composer_and_wandb.ipynb:1]: <p>it appears in the GitHub render</p><p> </p>

Comment (tcapelle) [reply colabs/mosaicml/MosaicML_Composer_and_wandb.ipynb:1]: done

Comment (tcapelle) [reply colabs/mosaicml/MosaicML_Composer_and_wandb.ipynb:1]: we do

**Pull request merged**